### DISCUSSION TITLE
Block chrome:// pages

### DISCUSSION THREAD

**Discussion by swiss-tim** [proginosko/LeechBlockNG#453] [Q&A] [answered]
The extension is fantastic. Is there a way to block Chrome/Edge pages, such as
chrome://extensions/?id=blaaajhemilngeeffpbfkdjjoefldkok? I could not find a way...

**Comment by proginosko** [comment, marked as answer]
Yes, this is possible using regular expressions. Click the "Show Advanced Options" button and enter the following into the field "Use regular expression to match URLs to block":

`^chrome:\/\/extensions\/\?id=blaaajhemilngeeffpbfkdjjoefldkok`

Be sure to enter it exactly as given; it's important to include the backslashes before the forward slashes and question mark.

Note that this will override any list of sites entered into the field under **What to Block**.
